packages/consumption/src/modules/requests/itemProcessors/transferFileOwnership/TransferFileOwnershipRequestItemProcessor.ts

```diff
@@ -145,33 +144,29 @@ export class TransferFileOwnershipRequestItemProcessor extends GenericRequestIte
             })
         });
 
-        const ownSharedIdentityAttribute = await this.consumptionController.attributes.createSharedLocalAttributeCopy({
-            sourceAttributeId: repositoryAttribute.id,
-            peer: requestInfo.peer,
-            requestReference: requestInfo.id
-        });
+        await this.consumptionController.attributes.addForwardedSharingDetailsToAttribute(ownIdentityAttribute, requestInfo.peer, requestInfo.id);
```

Comment:
This method does not have a return value .. why? Do you update the ownIdentityAttribute inPlace?